.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Node.js ${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: []
```

Comment:
What happened here?

Reply:
Initially, the script assumed the versions were only numbers. I later found that some projects used `stable`. Now, in production, I'm finding that some projects also used `iojs`.